roofit/roofitcore/src/TestStatistics/LikelihoodGradientWrapper.cxx

```diff
@@ -13,6 +13,11 @@
 #include "RooFit/TestStatistics/LikelihoodGradientWrapper.h"
 #include "RooMinimizer.h"
 
+// including derived classes for factory method
+#ifdef BUILD_WITH_ROOFIT_MULTIPROCESS
```

Comment:
Where are these macro defined ?
Would not be better to have something as in other ROOT libraries defined in RConfigure.h and with a name like: 
R__HAS_ROOFIT_MULTIPROCESS

Reply:
That's a good point! I have added a new commit with what you suggested.

Reply:
The macro existed already before this PR, by the way